edweiss-app/app/(app)/deck/[id]/index.tsx

```diff
@@ -115,12 +118,37 @@ const CardListScreen: ApplicationRoute = () => {
 		router.back();
 	}
 
+	const handleRefresh = () => {
+		setRefresh(true);
+	};
+
 	return (
 		<>
 			<RouteHeader
 				title={deck?.data.name}
 				right={
-					<Button color={'black'} testID='toggleButton' onPress={() => { setShowDropdown(true); modalRef_Operation.current?.present() }} title='⋮' />
+					<>
+						<TTouchableOpacity
+							testID='refreshButton'
+							onPress={handleRefresh}
+							activeOpacity={0.2}
+							backgroundColor={'transparent'}
+							mr={'md'}
+						>
+							<Icon name={'refresh'} size={30} />
+						</TTouchableOpacity>
+
+						<TTouchableOpacity
+							testID='toggleButton'
+							onPress={() => { setShowDropdown(true); modalRef_Operation.current?.present() }}
+							activeOpacity={0.2}
+							backgroundColor={'transparent'}
+						>
+							<Icon name={'trash'} size={30} />
+						</TTouchableOpacity>
+
+						{/*<Button color={'black'} testID='toggleButton' onPress={() => { setShowDropdown(true); modalRef_Operation.current?.present() }} title='⋮' />*/}
```

Comment:
Does this comment necessary? Or can we remove it?

Reply:
Oops, yes we can remove it. I was adapting the button to be TTouchableOpacity with Icon component. Fun fact, I was inspired by your code xD (you also use TTouchableOpacity and then Icon component in this way). Anyway, I tested and it is all good now so yeh, I can remove the commented code.